core/network/src/protocol.rs

```diff
@@ -670,6 +674,7 @@ impl<B: BlockT, S: NetworkSpecialization<B>, H: ExHashT> Protocol<B, S, H> {
 	/// Perform time based maintenance.
 	fn tick(&mut self) {
 		self.consensus_gossip.collect_garbage();
+		self.consensus_gossip.rebroadcast(&mut ProtocolContext::new(&mut self.context_data, &self.network_chan));
```

Comment:
This will happen every second which is probably too much. We should probably extract garbage collection and rebroadcasts to some other method and use a longer period.

Reply:
Agreed. Once every 10 or 30 seconds is probably better.